packages/angular_devkit/schematics/tools/workflow/node-workflow_spec.ts

```diff
@@ -14,7 +15,7 @@ import * as path from 'path';
 describe('NodeWorkflow', () => {
   // TODO: this test seems to either not work on windows or on linux.
   xit('works', done => {
-    const workflow = new NodeWorkflow(new NodeJsSyncHost(), { dryRun: true });
+    const workflow = new NodeWorkflow(new NodeJsSyncHost(), { dryRun: true, registry: new schema.CoreSchemaRegistry() });
```

Comment:
This test is disabled and appears broken.  The base workflow appears to create a default registry if not provided as well so I'd probably leave this one as is.